src/container/srv_container.c

```diff
@@ -5612,9 +5613,9 @@ ds_cont_op_handler(crt_rpc_t *rpc, int cont_proto_ver)
 
 		prop = cqo->cqo_prop;
 	} else if ((opc == CONT_OPEN) || (opc == CONT_OPEN_BYLABEL)) {
-		struct cont_open_out *coo = crt_reply_get(rpc);
+		struct cont_open_out *cout = crt_reply_get(rpc);
```

Comment:
No change requested. I guess it's OK for a C source code to name a variable to a C++ standard namespace object ("cout" here)? I guess compiling this with a C++ compiler will be fine? It would be a very unusual situation I suppose for daos_engine code to be compiled by a C++ compiler anyway.
Looks like this code was added in the initial incremental reintegration patch on master https://github.com/daos-stack/daos/pull/15671

Reply:
I forgot the cout is a c++ keyward, I will change it. the rename was suggested by the checkpatch, not intendend by myself.

Reply:
refreshed just to fix the naming.